### PR TITLE
Main readme PR

### PR DESCRIPTION
**Pull request by moratodpg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,100 @@
...
+- [**IQL**: Independent Q-Learning](https://arxiv.org/abs/1511.08779)
+
+The main code is derived from [PyMarl original implementation](https://github.com/oxwhirl/pymarl).
+
+## Expert-knowledge baselines available:
+- [Expert-based heuristic strategies](https://www.sciencedirect.com/science/article/pii/S0167473017302138)
+
+## Run an IMP environment 
+```
+import numpy
```
Comment (PaLeroy) [inline README.md]: Does this script needs Numpy?

File: README.md [hunk 2]
```diff
@@ -1,47 +1,100 @@
...
+
+# Simulate one episode selecting always the action do-nothing
+action_ = {
+    'agent_0' : 0,
+    'agent_1' : 0,
+    'agent_2' : 0,
+}
+done = False
+rewards_sum = 0
+while not done:
```
Comment (PaLeroy) [inline README.md:62]: Can we just have the loop?
I am not sure the content before provide more?
Here we need the smallest example as possible :D 

Like only:
config={aze}
Struct(config)
Structreset
while not done
action = random(action size)
step

File: README.md [hunk 3]
```diff
@@ -1,49 +1,43 @@
...
+![imp](imp_intro.png)
 
-If you will run experiments relying on pymarl, install the required packages:
-`./pymarl/install_venv.sh`
+## Main requirements:
+To work with our environments, one only needs to install [Numpy](https://numpy.org/install/).
 
-Otherwise, only numpy is required:
-`pip install numpy`
+However, to reproduce our results, more packages are required and installation instructions are provided [here](pymarl/README.md).
```
Comment (moratodpg) [inline README.md:22]: Very nice

File: README.md [hunk 4]
```diff
@@ -1,49 +1,43 @@
...
 
 ## Tutorials
-- [Create your own environment scenario](./imp_env/new_imp_env_tutorial.ipynb)
+- [Create your own environment scenario](imp_env/new_imp_env_tutorial.ipynb)
 - [IMP's API explained](imp_wrappers/wrapper_explained.md)
-- [Reproduce the reported results](./results_scripts/README.md)
-- [Retrieve directly the results](./results_scripts/README.md)
+- [Train agents like in the paper](pymarl/README.md)
+- [Reproduce the reported results](results_scripts/README.md)
```
Comment (moratodpg) [inline README.md]: This needs to be modified?

**Review by moratodpg — COMMENTED** [review]
Outstanding, only one minor comment

File: README.md (hunk 2 above)
Comment (PaLeroy) [reply README.md:62]: @moratodpg

File: README.md (hunk 1 above)
Comment (PaLeroy) [reply README.md]: @moratodpg

**Comment by PaLeroy** [comment]
I suggest this:
For me, this is self-explanatory. We do not need to import in a README and the lines are really easy to grasp, I feel adding comments is really not nice.


```
env = Struct({'n_comp': 3,
               'discount_reward': 0.95,
               'k_comp': 2,
               'env_correlation': False,
               'campaign_cost': False})

obs, rewards_sum, done = env.reset(), 0, False
while not done:
    actions = {f"agent_{i}": random.randint(0,2) for i in range(3)}
    obs, rewards, done, insp_outcomes = env.step(actions)
```

**Comment by moratodpg** [comment]
I am not sure everybody will be able to run this (without the importing packages). But I will change it.

**Pull request merged**